src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -92,7 +92,8 @@ public void getPersonList_modifyList_throwsUnsupportedOperationException() {
 
     @Test
     public void toStringMethod() {
-        String expected = AddressBook.class.getCanonicalName() + "{persons=" + addressBook.getPersonList() + "}";
+        String expected = AddressBook.class.getCanonicalName() + "{persons=" + addressBook.getPersonList() + ","
```

Comment:
You did not include relationships when turning the addressbook to string even though the addressbook contains relationships. Was this intentional?